torch/utils/tensorboard/writer.py

```diff
@@ -73,13 +74,24 @@ def __init__(self, log_dir, max_queue=10, flush_secs=120, filename_suffix=""):
         # TODO: See if we can remove this in the future if we are
         # actually the ones passing in a PosixPath
         log_dir = str(log_dir)
-        self.event_writer = EventFileWriter(
-            log_dir, max_queue, flush_secs, filename_suffix
-        )
+
+        self._log_dir = log_dir
+        self._max_queue = max_queue
+        self._flush_secs = flush_secs
+        self._filename_suffix = filename_suffix
+        self._event_writer = None
```

Comment:
```suggestion
        self._event_writer : Optional[EventFileWriter] = None
```
So it's not an implicit optional